docs/modules/con_thorntail-yaml-project-stages.adoc

```diff
@@ -0,0 +1,80 @@
+
+[id='the-thorntail-yaml-project-stages_{context}']
+= Configuration overlays using stage properties
+
+In some cases it may be desirable to extract some of the configuration properties and switch them depending on the target environment. Project stages are a way to move environment specific configuration properties into an external file.
```

Comment:
"switch _between_ them"?

Also I think the convention is 1 sentence = 1 line.

Reply:
It was a copy and paste from the old content, will update...

Reply:
I've kept a single sentence only...I'm not sure, when I read it again and again what the above sentence about the project stages means :-)

Reply:
Re the sentences, sorry I wasn't clear -- it's an docs authoring convention, to never wrap sentences in the middle and to never have more than one sentence on one line.